Mathlib/RingTheory/Ideal/Colon.lean

```diff
@@ -29,7 +29,7 @@ def colon (N P : Submodule R M) : Ideal R :=
 theorem mem_colon {r} : r ∈ N.colon P ↔ ∀ p ∈ P, r • p ∈ N :=
   mem_annihilator.trans
      ⟨fun H p hp => (Quotient.mk_eq_zero N).1 (H (Quotient.mk p) (mem_map_of_mem hp)),
-       fun H _ ⟨p, hp, hpm⟩ => hpm ▸ (N.mkQ.map_smul r p ▸ (Quotient.mk_eq_zero N).2 <| H p hp)⟩
```

Comment:
I wonder if this lemma was intended to be used in the reverse direction? It happens to be true by `rfl` so isn't necessary anyway, but might still be good to have here.

Reply:
I am of a mind to make the triangle rewrite in either direction, so maybe then the original code would have worked as intended? But does it matter that much if the present code works as well?